GliaWidgets/ViewController/Survey/Mocks/Survey.ViewController.Props.Mock.swift

```diff
@@ -0,0 +1,111 @@
+#if DEBUG
+import Foundation
+
+extension Survey.ViewController.Props {
+    static func emptyPropsMock() -> Survey.ViewController.Props {
+        return Survey.ViewController.Props(
+            header: "Survey title",
+            props: [
+                makeScalePropsMock(),
+                makeInputPropsMock(),
+                makeBooleanPropsMock(),
+                makeSinglePropsMock()
+            ],
+            submit: { _ in },
+            cancel: {}
+        )
+    }
+
+    static func filledPropsMock() -> Survey.ViewController.Props {
+        return Survey.ViewController.Props(
+            header: "Survey title",
+            props: [
+                makeScalePropsMock(selectedOption: .init(
+                    name: "1",
+                    value: 1
+                )),
+                makeInputPropsMock(value: ""),
+                makeBooleanPropsMock(selectedOption: .init(
+                    name: L10n.Survey.Action.yes,
+                    value: true
+                )),
+                makeSinglePropsMock(selectedOption: .init(
+                    name: "First option",
+                    value: "\(1)"
+                ))
+            ],
+            submit: { _ in },
+            cancel: {}
+        )
+    }
+}
+
+private extension Survey.ViewController.Props {
+
+    static func makeScalePropsMock(selectedOption: Survey.Option<Int>? = nil) -> Survey.ScaleQuestionView.Props {
+        var props = Survey.ScaleQuestionView.Props(
+            id: UUID().uuidString,
```

Comment:
Let's use `UUID.mock.uuidString` instead of `UUID().uuidString`

Reply:
bebee9d81b9be4257573913098767e71ba934270